src/pages/guidelines/content/glossary.mdx

```diff
@@ -7,4 +7,7 @@ tabs: ['General', 'Guidance', 'Glossary']
 
 import GlossaryComponent from 'components/GlossaryComponent';
 
+_Consistency across products plays a big role in the user experience.
+This glossary provides a lightweight reference for common UI terms and action labels._
+
```

Comment:
Can you update this to use our page description component so the styling is correct for the intro paragraph??

```
<PageDescription>

Consistency across products plays a big role in the user experience.
This glossary provides a lightweight reference for common UI terms and action labels

</PageDescription>
```